src/cli/commands/list.rs

```diff
@@ -0,0 +1,10 @@
+use std::path::PathBuf;
+
+use clap::Args;
+
+#[derive(Args, Debug)]
+pub struct List {
+	/// Root path
+	#[clap(short, long)]
```

Comment:
You can add a ValueHint: https://docs.rs/clap/latest/clap/enum.ValueHint.html
`DirPath` seems appropriate

Reply:
Yes that is planned, but I wanted to unblock you